libs/auth/src/drivers/interfaces/auth-query-manager.interface.ts

```diff
@@ -0,0 +1,17 @@
+import { MutationOptions } from 'apollo-client';
+import { DaffRegisterRequest } from '../../models/register-request';
+import { DaffCustomerRegistration } from '../../models/customer-registration';
+
+export interface DaffAuthQueryManagerInterface<
+  RegisterRequest extends DaffRegisterRequest<RegisterRequestT>,
+  RegisterRequestT extends DaffCustomerRegistration
+> {
+  generateATokenMutation(
+    email: string,
+    password: string
```

Comment:
I think this needs to be a generic object instead, because otherwise there would be no reason to ever override this service. Like:
```
generateATokenMutation(
   request: T
): MutationOptions<any>;
```
or something like that.